components/util/prefs.rs

```diff
@@ -14,9 +14,9 @@ use std::path::PathBuf;
 use std::sync::{Arc, Mutex};
 
 lazy_static! {
-    static ref PREFS: Arc<Mutex<HashMap<String, Pref>>> = {
+    pub static ref PREFS: Preferences = {
         let prefs = read_prefs().unwrap_or(HashMap::new());
-        Arc::new(Mutex::new(prefs))
+        Preferences(Arc::new(Mutex::new(prefs)))
```

Comment:
Actually, I'm wondering if we could use a `RWLock` instead of a `Mutex`, this should avoid locking the world when reading preferences, which is the common case.